Collections/Game.java

```diff
@@ -0,0 +1,53 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+
+enum Enum {
+    GENRE1, GENRE2, GENRE3

```

Comment:
Create a separate enum class. Enum must not include numbers in the value. Should then be `GENRE_ONE` `GENRE_TWO`

Reply:
Also enum class name should not be generic Enum. Something like Genre would suit more.

Reply:
ok